spec/models/maths_and_physics/dqt_hecos_code_record_check_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+RSpec.describe MathsAndPhysics::DQTRecord do
```

Comment:
It's not a big deal but I'd expect to see these specs in the model spec file at `spec/models/maths_and_physics/dqt_record_spec.rb`. 

Usually with unit tests the specs are found in a corresponding spec file.

Reply:
possibly confused myself with how it is on GH but is the file at the path you mentioned already?

Reply:
sorry, just got what you meant... yeah i originally had them in that file and then decided to separate it out because there was quite a few of them.

Reply:
These specs are in `spec/models/maths_and_physics/dqt_hecos_code_record_check_spec.rb` at the moment. I'd usually expect to see them in `spec/models/maths_and_physics/dqt_record_spec.rb`. Happy to go through it on Slack if you like.

Reply:
Ah I see... I think it's fine to have long test files. The specs act as documentation for our model so future developers will probably expect to see them with all the other model specs.

Also there's some developer tooling that expect unit tests to be named this way. For example in RubyMine when I press shift+cmd+t I can toggle between model and test files instantly.

Reply:
That makes sense for folder/file structure and naming to take precedence 

ahhh, that's very cool RubyMine does that, i'd never heard of it before so will check it out.